components/slices/Process.vue

```diff
@@ -0,0 +1,339 @@
+<template>
+  <section class="process">
+    <prismic-rich-text v-if="hasTitle" class="process__title" :field="title" />
+    <prismic-rich-text
+      v-if="hasSubtitle"
+      class="process__subtitle"
+      :field="subtitle"
+    />
+    <div class="process__wrapper" :style="[background]">
+      <div
+        v-for="(itemsInARow, rowIndex) in itemsByRow"
+        :key="`item-${rowIndex}`"
+        class="process-wrapper__row"
+      >
+        <div
+          v-for="(item, itemIndex) in itemsInARow"
+          :key="`item-${itemIndex}`"
+          class="process-wrapper__item"
+        >
+          <prismic-image v-if="hasImage(item)" :field="item.item_image" />
+          <div>
+            <prismic-rich-text
+              class="process-wrapper-item__title"
+              :field="item.item_title"
+            />
+            <prismic-rich-text
+              class="process-wrapper-item__description"
+              :field="item.item_description"
+            />
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+</template>
+
+<script>
+import {
+  DESKTOP_MIN_WIDTH,
+  EXTRA_LARGE_SCREEN_MIN_WIDTH,
+  LARGE_SCREEN_MIN_WIDTH,
+  TABLET_MIN_WIDTH,
+} from '~/config/breakpoints'
+
+export default {
+  name: 'ProcessSlice',
+  props: {
+    slice: {
+      type: Object,
+      default: null,
+    },
+  },
+  data() {
+    return {
+      itemsByRow: [],
+    }
+  },
+  computed: {
+    background() {
+      return {
+        backgroundColor: `${this.slice.primary.process_background_color}`,
+      }
+    },
+    items() {
+      return this.slice.items
+    },
+    hasTitle() {
+      return (
+        this.slice.primary.process_title &&
+        this.slice.primary.process_title.length &&
+        this.slice.primary.process_title[0].text &&
+        this.slice.primary.process_title[0].text.length
+      )
```

Comment:
```suggestion
      return this.slice.primary.process_title.length && this.slice.primary.process_title[0].text.length    
```
devrait suffire non ?  🤔